src/api/apiFetch.js

```diff
@@ -40,7 +40,8 @@ export const apiFetch = async (
   auth: Auth,
   route: string,
   params: $Diff<$Exact<RequestOptions>, {| headers: mixed |}>,
-) => fetch(new URL(`/${apiVersion}/${route}`, auth.realm).toString(), getFetchParams(auth, params));
+): Promise<empty> =>
```

Comment:
This one actually doesn't need to be exported at all! So that's the simplest fix for this one.